src/main/java/org/hobbit/core/components/AbstractCommandReceivingComponent.java

```diff
@@ -253,9 +254,20 @@ protected String createContainer(String imageName, String[] envVariables) {
      */
     protected String createContainer(String imageName, String containerType, String[] envVariables) {
         try {
-            envVariables = envVariables != null ? Arrays.copyOf(envVariables, envVariables.length + 2) : new String[2];
-            envVariables[envVariables.length - 2] = Constants.RABBIT_MQ_HOST_NAME_KEY + "=" + rabbitMQHostName;
+            if (envVariables == null) {
+                envVariables = new String[0];
```

Comment:
Could you please directly initialize it with the two constants you would like to define?

After that, the if should also get an else block which contains the rest of the workflow until line 270.